### PR TITLE
Update bmv2 README with development build instructions

### PR DESCRIPTION
**Pull request by bocon13**

### CODE REVIEW

**Comment by bocon13** [comment]
@donNewtonAlpha here are some updated instructions that might make incremental builds easier to test

File: stratum/hal/bin/bmv2/README.md [hunk 1]
```diff
@@ -25,16 +29,24 @@ setup_dev_env.sh  # You're inside the Docker container now
...
-installed as from a Debian package:
+The `stratum_bmv2` binary can be run directly after installing the
+Debian package:
 
 ```bash
-# Or just `stratum_bmv2`
-./bazel-bin/stratum/hal/bin/bmv2/stratum_bmv2 \
-    --chassis_config_file=/etc/stratum/chassis_config.pb.txt \
-    --bmv2_log_level=debug
+dpkg -i stratum_bmv2_deb.deb
```
Comment (pudelkoM) [inline stratum/hal/bin/bmv2/README.md:46]: `apt-get` for the deps

Comment (bocon13) [reply stratum/hal/bin/bmv2/README.md:46]:
```suggestion
[sudo] apt-get update
[sudo] apt-get install -y --reinstall ./stratum_bmv2_deb.deb
```

File: stratum/hal/bin/bmv2/README.md [hunk 2]
```diff
@@ -69,5 +82,136 @@ cp stratum/hal/bin/bmv2/update_config.py /tmp/ && \
...
+export BMV2_DIR=$(bazel info workspace)/stratum/hal/bin/bmv2
+bazel build //stratum/hal/bin/bmv2:stratum_bmv2
+sudo bazel-bin/stratum/hal/bin/bmv2/stratum_bmv2 \
+    -persistent_config_dir=/tmp/ \
+    -chassis_config_file=${BMV2_DIR}/chassis_config.pb.txt \
+    -initial_pipeline=${BMV2_DIR}/dummy.json \
+    -forwarding_pipeline_configs_file=/tmp/bmv2_pipeline_cfg \
+    -bmv2_log_level=debug
+```
+
```
Comment (donNewtonAlpha) [inline stratum/hal/bin/bmv2/README.md:155]:
```
I20210802 18:56:46.491569   408 p4_service.cc:121] Pushing the saved forwarding pipeline configs read from /tmp/bmv2_pipeline_cfg...
E20210802 18:56:46.491609   408 utils.cc:112] StratumErrorSpace::ERR_FILE_NOT_FOUND: /tmp/bmv2_pipeline_cfg not found.
E20210802 18:56:46.491842   408 utils.cc:68] Return Error: ReadFileToString(filename, &text) failed with StratumErrorSpace::ERR_FILE_NOT_FOUND: /tmp/bmv2_pipeline_cfg not found.
W20210802 18:56:46.491852   408 p4_service.cc:130] No saved forwarding pipeline config found at /tmp/bmv2_pipeline_cfg. This is normal when the switch is just installed and no master controller is connected yet. 
``` 
I assume this is expected - is there a canned config for bmv2 that can be saved to /tmp ? or pushed for that matter?

File: stratum/hal/bin/bmv2/README.md [hunk 3]
```diff
@@ -69,5 +82,136 @@ cp stratum/hal/bin/bmv2/update_config.py /tmp/ && \
...
+```bash
+sudo veth_setup.sh
+```
+
+When you are finished, you can remove the interfaces.
+```bash
+sudo veth_teardown.sh
+```
+
+If you don't have the `veth_setup.sh` command installed, you can
```
[the remark this answers is not in the document]
Comment (pudelkoM) [reply stratum/hal/bin/bmv2/README.md:118]: https://github.com/p4lang/behavioral-model/blob/main/tools/veth_setup.sh

File: stratum/hal/bin/bmv2/README.md (hunk 2 above)
Comment (pudelkoM) [reply stratum/hal/bin/bmv2/README.md:155]: I think fabric-tna should be pushable to bmv2: https://github.com/stratum/fabric-tna

**Review by pudelkoM — DISMISSED** [review]

File: stratum/hal/bin/bmv2/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bocon13) [reply stratum/hal/bin/bmv2/README.md:118]: Added link

**Pull request merged**